public/embeddable/map_component.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React, { useEffect, useState } from 'react';
+import {
+  withEmbeddableSubscription,
+  EmbeddableOutput,
+} from '../../../../src/plugins/embeddable/public';
+import { MapEmbeddable, MapInput } from './map_embeddable';
+import { MapComponent } from '../components/map_page/';
+import { OpenSearchDashboardsContextProvider } from '../../../../src/plugins/opensearch_dashboards_react/public';
+import { MapServices } from '../types';
+import { TimeRange } from '../../../../src/plugins/data/common';
+
+interface Props {
+  embeddable: MapEmbeddable;
+  input: MapInput;
+  output: EmbeddableOutput;
```

Comment:
do we need this property?

Reply:
Yes, that's required from embeddable plugin